apps/anon-message-client/.env.local.example

```diff
@@ -0,0 +1,10 @@
+# The port on which this server runs. We recommend `4000` for convenient
+# local integration with the rest of the services that run within this
+# repository.
+NEXT_PUBLIC_PORT=4000
```

Comment:
it looks like we use `process.env.PORT` [here](https://github.com/proofcarryingdata/zupass/pull/680/files#diff-7aaae75bbe81b4e4fc4340cca94ad4f98f1354a0895de79ff01a1f264b909316R6) and we hard code 4000 in `yarn start`. you should either hardcode it to 4000 everywhere and remove the environment variable, or keep it consistent naming-wise

Reply:
Will remove the hardcoded from `yarn start`. Prepending env vars with `NEXT` is required for env vars in the client for Next.js. It will still read as `PORT` when used though. It's weird.